app/helpers/rails_db/tables_helper.rb

```diff
@@ -43,5 +43,17 @@ def table_pagination_path
       params.merge({action: :data})
     end
 
+    def column_is_checked?(column_name)
+      if cookies["Table: #{params[:table_id]}"].present?
+        !cookies["Table: #{params[:table_id]}"].include?(" #{column_name}")

```

Comment:
BAD CODE

what if you have column with name "user_id" and "old_user_id" ? or "user_id_old" ??? Rewrite this method, use split